.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,29 @@
+# Upload a Python package when a release is created
+# https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows
+
+name: Publish Python 🐍 distributions 📦 to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+
+      - name: Build a source tarball and a binary wheel
+        # https://pypa-build.readthedocs.io
+        run: |
+          python -m pip install build
+          python -m build --sdist --wheel
+
+      - name: Publish 📦 to PyPI
+        # https://github.com/pypa/gh-action-pypi-publish
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
I assume the secret is set on org level, but we should verify that just to make sure.

Reply:
As far as I can tell, it's not, I actually set my own token on repo level.

Reply:
It's a temporary one anyway, as the project on PyPI doesn't exist yet.

Reply:
Oh, I remember now, the token is not tied to an account but rather a project IIRC. All good then :)